src/components/admin/navigation/adminMenu.tsx

```diff
@@ -4,7 +4,7 @@ import {
   People,
   ContactPhone,
   Payment,
-  CoPresent,
```

Comment:
The reason for that icon change is that I had an error that the icon couldn't be found

Reply:
The `CoPresent` icon is part of the updated mui icons package, you need to install dependencies and it will be fine

https://mui.com/components/material-icons/?query=copresent&selected=CoPresent